oauth2/oauth2-resource-server/src/main/java/org/springframework/security/oauth2/resourceserver/authentication/JwtAccessTokenVerifier.java

```diff
@@ -13,7 +13,6 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-
```

Comment:
There are no changes in this file?

Reply:
Probably my formatter removing a line between the copyright and the package. I'll take it out of the PR.